### ISSUE TITLE
Hover info doesn't increment past the shortest dataset's max X value

### ISSUE THREAD

**Issue by badair** [ldionne/metabench#158]
![chart_point](https://cloud.githubusercontent.com/assets/2570307/20236488/43b7b2d6-a87c-11e6-828b-2cdc0278f446.png)

The hover info in this screenshot shows 280, even though the point is at the 300 vertical. This might be a bug in a different project, I don't know. Either way, it's really minor. [live example(http://output.jsbin.com/xaxiguriqi)

**Comment by ldionne** [comment]
I'm not sure whether it's actually a bug. The thing is that you're really looking at two different points, one on the blue curve and the other one on the orange curve. These two points have a different X value, but the tooltip is only showing one X value. Which one should it show? Should it stop showing the blue point when you're out of range? Also note that the Y value of the orange curve is updating properly, i.e. it does not stay stuck at the X=280.

**Comment by badair** [comment]
> Which one should it show? Should it stop showing the blue point when you're out of range?

Yes, I would think so. I think ideally the X axis value would update to the hovered point, and the datasets that do not cross the vertical would disappear. I guess I wouldn't spend my free time changing it (it's already far more polished than I would expect), but I wanted to bring it to your attention in case it's something you'd want to change.
